server/boot/z-book-people.js

```diff
@@ -0,0 +1,37 @@
+var async = require('async');
+
+module.exports = function(app) {
+  var Book = app.models.Book;
+  var Author = app.models.Author;
+  var Reader = app.models.Reader;
+
+  var book;
+  var author;
+  var reader;
+  async.series([
+    function createAuthor(done) {
+      Author.create({ name: 'Author 1' }, function(err, p) {
```

Comment:
Be consistent with your space around inline object declarations: ie: here you have `{ name... }` and in the later code, you have `{notes: ...}`. I suggest removing the whitespace at the beginning and end (ie. use the `{notes: 'Note 1'}` style.